Explain documentation versions and relationship with branches (#1106)
PrestaShop/docs: contribute/documentation/how/documentation-versions.md

---
title: Documentation versions
weight: 8
---

# Documentation versions

The documentation content is tracked in the [Docs Repository](https://github.com/PrestaShop/docs). Different versions of the documentation are available, that match the major versions of PrestaShop.

Contributions to the content, such as adding a new page, updating or fixing some existing reference, must be submitted as a Pull Request against the right branch on the repository.

> PrestaShop 1.7.x documentation is tracked in the `1.7.x` branch.
> 
> PrestaShop 8.x documentation is tracked in the `8.x` branch.

Reviewer: ```suggestion
- PrestaShop 1.7.x documentation is tracked in the `1.7.x` branch.
- PrestaShop 8.x documentation is tracked in the `8.x` branch.
```

> Consequently if you want to update the 8.x documentation you must submit your Pull Request against the `8.x` branch.

Reviewer: ```suggestion
Consequently, if you want to update the 8.x documentation, you must submit your Pull Request against the `8.x` branch.
```

> If your contribution should be merged for multiple documentation versions, we suggest to first submit it against the highest version, then cherry-pick to previous versions.

Reviewer: ```suggestion
If your contribution is applicable for multiple PrestaShop versions, we suggest submitting it against the highest version first, then backporting the changes into previous versions in independent Pull Requests.
```
